torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -222,6 +222,16 @@ struct SourceImporterImpl : public Resolver,
       return std::make_shared<SimpleValue>(
           graph->insertConstant(std::numeric_limits<double>::quiet_NaN(), loc));
     }
+    if (name == "infj") {
+      return std::make_shared<SimpleValue>(graph->insertConstant(
+          c10::complex<double>(0, std::numeric_limits<double>::infinity()),
+          loc));
+    }
+    if (name == "nanj") {
+      return std::make_shared<SimpleValue>(graph->insertConstant(
+          c10::complex<double>(0, std::numeric_limits<double>::quiet_NaN()),
+          loc));
+    }
```

Comment:
It seems weird to me that these constants would be present in the serialized code as `infj` and `nanj` without any qualifying namespace or module like `cmath`. Is that because of how complex numbers are printed (i.e. `[imaginary part]j`)? Can you include a snippet of serialized code that contains complex constants (just for my own education)?

Reply:
Python Code:
```
In [12]: def fn():
    ...:     return cmath.infj
    ...:

In [13]: torch.jit.save(torch.jit.script(fn), "infj.pt")
```

Serialized code:
```
class PlaceholderModule(Module):
  __parameters__ = []
  __buffers__ = []
  training : bool
  def forward(self: __torch__.PlaceholderModule) -> complex:
    return 0.+infj
```